src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -106,49 +106,78 @@ export default class CoveredEResourcesList extends React.Component {
     date ? <FormattedUTCDate value={date} /> : '-'
   )
 
-  renderExportDropdown = () => (
-    <Dropdown
-      onToggle={() => this.setState(prevState => ({ exportDropdownOpen: !prevState.exportDropdownOpen }))}
-      open={this.state.exportDropdownOpen}
+  renderDropdownButton = (disabled) => (
+    <DropdownButton
+      data-role="toggle"
+      disabled={disabled}
     >
-      <DropdownButton data-role="toggle">
-        <FormattedMessage id="ui-agreements.eresourcesCovered.exportAs" />
-      </DropdownButton>
-      <DropdownMenu data-role="menu">
-        <FormattedMessage id="ui-agreements.eresourcesCovered.exportAsJSON">
-          {exportAsJson => (
-            <Button
-              aria-label={exportAsJson}
-              buttonStyle="dropdownItem"
-              id="clickable-dropdown-export-eresources-json"
-              onClick={() => {
-                this.setState({ exportDropdownOpen: false });
-                this.props.onExportEResourcesAsJSON();
-              }}
-            >
-              <FormattedMessage id="ui-agreements.eresourcesCovered.json" />
-            </Button>
-          )}
-        </FormattedMessage>
-        <FormattedMessage id="ui-agreements.eresourcesCovered.exportAsJSON">
-          {exportAsKbart => (
-            <Button
-              aria-label={exportAsKbart}
-              buttonStyle="dropdownItem"
-              id="clickable-dropdown-export-eresources-kbart"
-              onClick={() => {
-                this.setState({ exportDropdownOpen: false });
-                this.props.onExportEResourcesAsKBART();
-              }}
-            >
-              <FormattedMessage id="ui-agreements.eresourcesCovered.kbart" />
-            </Button>
-          )}
-        </FormattedMessage>
-      </DropdownMenu>
-    </Dropdown>
+      <FormattedMessage id="ui-agreements.eresourcesCovered.exportAs" />
+    </DropdownButton>
   )
 
+  renderExportDropdown = () => {
+    const disabled = this.props.eresourcesFilterPath === 'dropped' || this.props.eresourcesFilterPath === 'future';
+    return (
+      <Dropdown
+        onToggle={() => this.setState(prevState => ({ exportDropdownOpen: !prevState.exportDropdownOpen }))}
+        open={this.state.exportDropdownOpen}
+      >
+        {disabled ?
+          <Tooltip
+            id="covered-eresources-export-tooltip"
+            placement="top"
+            text={<FormattedMessage id="ui-agreements.eresourcesCovered.exportButton.tooltip" />}
+          >
+            {({ ref, ariaIds }) => (
+              <div
+                ref={ref}
+                aria-labelledby={ariaIds.text}
```

Comment:
Additionally, you should [set the `aria-describedby` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute) and not `aria-labelledby`. You're describing the button with this tooltip, not labelling it; the thing that labels the button is the _actual_ label of the button.